WalletWasabi.Fluent/ApplicationStateManager.cs

```diff
@@ -90,14 +92,14 @@ internal ApplicationStateManager(IClassicDesktopStyleApplicationLifetime lifetim
 			.Permit(Trigger.MainWindowClosed, State.Closed)
 			.OnTrigger(Trigger.Show, MainViewModel.Instance.ApplyUiConfigWindowSate);
 
-		_lifetime.ShutdownRequested += LifetimeOnShutdownRequested;
-
 		_stateMachine.Start();
-
-		if (!startInBg)
+		if (_openInBg)
 		{
-			_stateMachine.Fire(Trigger.Loaded);
```

Comment:
> This fixes the bug when on startup the wallet didn't start hidden.

If this is the goal. Wouldn't have been enough to change the condition and the trigger here?
I don't even understand the current condition. It would be better like this:

```c#
if (startInBg)
{
	_stateMachine.Fire(Trigger.Hide);
}
```

Because the statemachine will start in Open state because of `	.InitialTransition(State.Open)` line, so if we want to have it in the backgound, the Hide trigger should be enough in the case of startInBg is true.

Reply:
> Wouldn't have been enough to change the condition and the trigger here?

This is kinda what I did, but `_hideRequest = true;` is needed, otherwise the wallet gets into a bugged state where it can't be closed because shutdown preventiont turns on, and `_openInBg = false;` is for that first time show trigger prevention I mentioned in my [other comment](https://github.com/zkSNACKs/WalletWasabi/pull/10123#discussion_r1109608033), which I totally agree is weird and probably not the way to go.

> Because the statemachine will start in Open state because of 	.InitialTransition(State.Open) line,

I didn't know that, I was looking for an hour in code why the wallet pops up every time even if I don't call show.
If this is why, the better approach would be to set initialTransition to `State.Closed`.

Reply:
Simplified it